sdk/identity/identity/karma.conf.js

```diff
@@ -0,0 +1,149 @@
+    envPreprocessor: ["ACCOUNT_NAME", "ACCOUNT_SAS", "TEST_MODE"],
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ["mocha", "coverage", "remap-coverage", "junit", "json-to-file"],
+
+    coverageReporter: { type: "in-memory" },
+
+    // Coverage report settings
+    remapCoverageReporter: {
+      "text-summary": null, // to show summary in console
+      html: "./coverage-browser",
+      cobertura: "./coverage-browser/cobertura-coverage.xml"
+    },
+
+    // Exclude coverage calculation for following files
+    remapOptions: {
+      exclude: /node_modules|test/g
+    },
+
+    junitReporter: {
+      outputDir: "", // results will be saved as $outputDir/$browserName.xml
+      outputFile: "test-results.browser.xml", // if included, results will be saved as $outputDir/$browserName/$outputFile
+      suite: "", // suite will become the package name attribute in xml testsuite element
+      useBrowserName: false, // add browser name to report and classes names
+      nameFormatter: undefined, // function (browser, result) to customize the name attribute in xml testcase element
+      classNameFormatter: undefined, // function (browser, result) to customize the classname attribute in xml testcase element
+      properties: {} // key value pair of properties to add to the <properties> section of the report
+    },
+
+    jsonToFileReporter: {
+      filter: function(obj) {
+        if (obj.writeFile) {
+          const fs = require("fs-extra");
+          // Create the directories recursively incase they don't exist
+          try {
+            // Stripping away the filename from the file path and retaining the directory structure
+            fs.ensureDirSync(obj.path.substring(0, obj.path.lastIndexOf("/") + 1));
+          } catch (err) {
+            if (err.code !== "EEXIST") throw err;
+          }
+          fs.writeFile(obj.path, JSON.stringify(obj.content, null, " "), (err) => {
+            if (err) {
+              throw err;
+            }
+          });
+        }
+        return false;
+      },
+      outputPath: "."
+    },
+
+    // web server port
+    port: 9876,
+
+    // enable / disable colors in the output (reporters and logs)
+    colors: true,
+
+    // level of logging
+    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+    logLevel: config.LOG_INFO,
+
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: false,
+
+    // start these browsers
+    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
+    // 'ChromeHeadless', 'Chrome', 'Firefox', 'Edge', 'IE'
+    browsers: ["ChromeHeadless"],
+
+    // Continuous Integration mode
+    // if true, Karma captures browsers, runs the tests and exits
+    singleRun: false,
```

Comment:
Shouldn't this be true? Since we're running these tests in pipelines.

Reply:
Copypasta from the storage libraries.  I'm using the `--single-run` parameter in the `npm` script but it's probably better to just set this setting to true.  Thanks!

Reply:
Hmmm.